cdc/entry/mounter.go

```diff
@@ -272,6 +272,7 @@ const (
 )
 
 // UnmarshalDDL unmarshals the ddl job from RawKVEntry
+// todo (Ling Jin): how to determine it's a valid ddl event
```

Comment:
```suggestion
// TODO(Ling Jin): how to determine it's a valid ddl event
```

What do you mean by "valid"?